src/main/java/cashleh/transaction/IncomeCategories.java

```diff
@@ -9,6 +9,13 @@ public enum IncomeCategory implements Categories {
         OTHERS
     }
 
+    public static Categories getCategory(String categoryString) {
+        try {
+            return ExpenseCategories.ExpenseCategory.valueOf(categoryString.toUpperCase());
+        } catch (IllegalArgumentException e) {
+            return ExpenseCategories.ExpenseCategory.OTHERS;
+        }
+    }
```

Comment:
im unsure why there is expenseCategory involved in the code for incomeCatgeories

Reply:
paiseh this method not required i forgot to remove